docs/rfcs/accepted/0002-consolidate_svp_and_crypto_cipher_functions/README.md

```diff
@@ -16,8 +18,6 @@ to the SecApi 3 library. The intention is that the SecApi 2 Adapter library can
 with a new SecApi 3 library and can provide compatability for applications and libraries that have
 not upgraded to the new interface yet.
 
-![SecApi3-Plan](../diagrams/SecApi3-Plan.png)
```

Comment:
Why remove this? It was part of the original RFC.

Reply:
I think we should fix the image location to be correct in both disk and rendered HTML for historical reasons.